code/bit-manipulation/thrice_unique_number/thrice_unique_number.c

```diff
@@ -0,0 +1,17 @@
+#include<stdio.h>
```

Comment:
The solution is expected to use bit manipulation. Kindly take a look at [this C++ code](https://github.com/OpenGenus/cosmos/blob/master/code/bit-manipulation/thrice_unique_number/threeUnique.cpp) and rewrite this 👍